example/lib/main.dart

```diff
@@ -53,6 +58,24 @@ class _MyAppState extends State<MyApp> {
     initPlatformState();
     activateCleverTapFlutterPluginHandlers();
     CleverTapPlugin.setDebugLevel(3);
+    if (kIsWeb) {
+      CleverTapPlugin.init("WRK-485-456Z", "sk1-staging-4", "wzrkt.com");
```

Comment:
@KambleSonam Remove the Account ID and Region and change it to `YOUR_ACCOUNT_ID` and `YOUR_ACCOUNT_REGION`